scripts/sync-to-downstream.sh

```
@@ -0,0 +1,503 @@
+
+  if [ ! -d "$consumer_full" ]; then
+    REPLY="missing"
+    return 0
+  fi
+
+  local drift_count=0
+  local missing_count=0
+  # Walk every file under the Mergepath kit. Ignore symlinks and
+  # special files — kits are plain script directories.
+  while IFS= read -r -d '' f; do
+    local rel="${f#$mp_full/}"
```

Shellcheck (0.11.0)

[info] Expansions inside ${..} need to be quoted separately, otherwise they match as patterns.

(SC2295)